pkg/yurttunnel/server/interceptor.go

```diff
@@ -39,8 +40,23 @@ var (
 	supportedHeaders       = []string{constants.ProxyHostHeaderKey, "User-Agent"}
 	HeaderTransferEncoding = "Transfer-Encoding"
 	HeaderChunked          = "chunked"
+	bufioReaderPool        sync.Pool
 )
 
+func newBufioReader(r io.Reader) *bufio.Reader {
```

Comment:
please add comments for newBufioReader func.

Reply:
done.